db/fixtures/customization_templates.yml

```diff
@@ -308,226 +308,3 @@
     - nmcli connection up eth0
   :type: CustomizationTemplateCloudInit
   :system: true
-
```

Comment:
@pkliczewski IIRC, users can't select the system templates, they are just there as examples.  If desired, the admin can copy and customize them.

Reply:
@bdunne Customization of a template is one thing and provide params to run it is completely different.